### ISSUE TITLE
Not consistant logging across Hosting

### ISSUE THREAD

**Issue by Muchaszewski** [dotnet/aspnetcore#5910]
Following files and methods contains `Console.Write` or `Console.WriteLine` instead of using ILogger interface. That information, though informational might be useful in debugging process on remote host where we do not read console directly.

Following files should have replaced those `Console.Write` with `_logger.LogInformation(...)`:
`src/Microsoft.Extensions.Hosting/Internal/ConsoleLifetime.cs:35-37`
`src/Microsoft.AspNetCore.Hosting/WebHostExtensions.cs:100,101,109`

Following files should have replaced those `Console.Write` with `_logger.LogError(...)`:
`src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs:163,168`
`src/Microsoft.AspNetCore.Hosting/Internal/WebHost.cs:226`

Following files and lines were provided from commit: `9f4c7c172f4876db77cc3b4e1eef50e61608ac60`

**Comment by gthvidsten** [comment]
I've just encountered this issue and found it very confusing that I set up logging, but the startup messages for `WebHost` only appeared in the console and not in the logs.

`WebHost` should not behave radically different than a generic host. A generic host logs startup messages to the logger you define, but `WebHost` does not (as evidenced by the [source code](https://github.com/aspnet/AspNetCore/blob/71c5c66b211c7ab9d44751e39cef27c525124bb7/src/Hosting/Hosting/src/WebHostExtensions.cs#L116)).

I would strongly suggest making a `WebHost` output startup messages to the logger instead of to the console. First and foremost for consistency across various hosts, but also to prevent confusion about messages not appearing in any logs.
